crates/core/src/contract/mod.rs

```diff
@@ -228,6 +228,26 @@ where
                         tracing::debug!(%error, "shutting down contract handler");
                     })?;
             }
+            ContractHandlerEvent::QuerySubscriptions { callback } => {
```

Comment:
I don't think is where you want to get this information from, instead it should be coming from the OperationManager/Ring type, which is available in p2p_protoc, this are "applications" subscribing to a cotnract, not remote peers

Maybe this information is also useful for debugging but is not the one you are looking for here